### PR TITLE
Test part 1

### PR DESCRIPTION
**Pull request by ForHemer**
In this milestone, my coding partner @alvarorf  and I have created successful tests for the functions responsible for adding and deleting list items.

Screenshot from 2022-02-25 08-11-24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ TO-DO LIST is a basic website that allows users to add/remove task from a to-do
 
 ## Screenshot
 
-![Screenshot](https://user-images.githubusercontent.com/88809610/154808929-c35b5e17-57a9-4757-a232-d576ae57b01a.png)
+![Screenshot from 2022-02-24 23-41-47](https://user-images.githubusercontent.com/88809610/155655442-dd097259-bfb1-43ce-bf22-49fa41daddcf.png)
```
Comment (medaminedev66) [inline README.md:16]: - [ ] Let's add some steps to the `Usage` section in README of how to test the project locally (Commands that the user should write in the Terminal to test your project) so your project is documented professionally.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ForHemer and @alvarorf, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ForHemer) [reply README.md:16]: > * [ ]  Let's add some steps to the `Usage` section in README of how to test the project locally (Commands that the user should write in the Terminal to test your project) so your project is documented professionally.

Thank you @medaminedev66 for your suggestion, Everything is ok now.

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @ForHemer , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**